### PR TITLE
Potifolio-collab Add-clean-code

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
cleaned codebase

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (kinginthenorthcodez) [inline .github/workflows/linters.yml:48]: Please remember to check your codebase is linter error-free.
#You can run to install:
- npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x `stylelint-csstree-validator@1.x`

File: css/style.css [hunk 2]
```diff
@@ -1,68 +1,109 @@
...
+#projects {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: center;
 }
 
-.project-title{
```
Comment (kinginthenorthcodez) [inline css/style.css:42]: please remove any unused code like here

File: index.html [hunk 3]
```diff
@@ -2,37 +2,46 @@
...
   </head>
 
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 <navbar id="navbar">
-  <ul>
```
Comment (kinginthenorthcodez) [inline index.html:14]: please remove unused code and clean your codebase

File: index.html [hunk 4]
```diff
@@ -2,37 +2,46 @@
...
-  <a href="www.google.com">Mygoogle</a>
-
+   <ul class="social-links">
+     <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i> Github </a></li>
+     <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a></li>
+     <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i> Linkedin</a></li>
+   </ul>
 </footer>
 <script  src="js/index.js"></script>
 </body>
```
Comment (kinginthenorthcodez) [inline index.html:47]: please check for html close tag missing. It is best practice to close element tags

**Review by kinginthenorthcodez — COMMENTED** [review]
Changes required ♻️ 
>Please make some changes as required in the comments 👍🏽 
- [ ✔️ ]  Correct Github flow used
- [✔️  ]  The repository is documented in a professional way

- [✔️ ] Apply best practices in HTML code.
- [✔️  ] Use semantic HTML tags.
- [✔️ ] Use CSS selectors correctly.
- [ ✔️ ] Use CSS box model.

**Review by ReshyResh — APPROVED** [review]
# Hi @kinginthenorthcodez , 

### Your project is complete! 

  - ✔Good explanation as for why a change is required.
  - ✔Polite comments and suggestions rather than orders.
  - ✔Clear and descriptive messages.


#### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by kinginthenorthcodez** [comment]
Thanks for your review

**Pull request merged**